doc/getting-started/overview_dev.rst

```diff
@@ -82,13 +83,12 @@ that ACRN offers:
 
 ACRN offers three types of VMs:
 
-* **Pre-launched User VMs**: These VMs are automatically launched at boot time
-  by the hypervisor. They run independently of other VMs and own dedicated
-  hardware resources, such as a CPU core, memory, and I/O devices. Other VMs,
-  including the Service VM, may not even be aware of a pre-launched VM's
-  existence. The configuration of pre-launched VMs is static and must be defined
-  at build time. They are well-suited for safety-critical applications and where
-  very strict isolation, including from the Service VM, is desirable.
+* **Pre-launched User VMs**: These VMs run independently of other VMs and own
+  dedicated hardware resources, such as a CPU core, memory, and I/O devices.
```

Comment:
```suggestion
  dedicated hardware resources, such as a CPU cores, memory, and I/O devices.
```

Reply:
That's mixing singular and plural, isn't it? "a ... cores"